tests/integration/test_storage_kafka/configs/kafka.xml

```diff
@@ -11,16 +10,39 @@
         <debug>cgrp,consumer,topic,protocol</debug>
 
         <!-- librdkafka stat in system.kafka_consumers -->
-				<!-- default 3000 (every three second) seems too long for test -->
-				<statistics_interval_ms>600</statistics_interval_ms>
+        <!-- default 3000 (every three second) seems too long for test -->
+        <statistics_interval_ms>600</statistics_interval_ms>
 
         <kafka_topic>
             <name>consumer_hang</name>
             <!-- default: 3000 -->
             <heartbeat_interval_ms>300</heartbeat_interval_ms>
+        </kafka_topic>
+        <consumer>
+          <auto_offset_reset>earliest</auto_offset_reset>
+            <kafka_topic>
+            <!-- Setting for topic will be applied only for consumer -->
+            <name>consumer_hang</name>
```

Comment:
Please intend this, it's visually hard to keep things apart.

Reply:
Sorry for this. Fixed.